Small improvements to the atomicradio Presence

VocalZero opened the pull request.

InstabilSpielt commented:
We have made minor changes, including moving the status page so that information can still be given to our users in the event of an incident. In addition, the documentation for developers has been released and has been integrated and the logo is now delivered in the most optimal size.

Bas950 commented on websites/A/atomicradio/dist/metadata.json:
```diff
@@ -1,23 +1,32 @@
-    "webradio"
-  ],
-  "category": "music"
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
```
use latest schema (v1.3)

Bas950 commented on websites/A/atomicradio/dist/metadata.json:
```diff
@@ -1,23 +1,32 @@
+        "atomicradio.eu",
+        "status.atomicradio.eu",
+        "docs.atomicradio.eu"
+    ],
+    "version": "1.0.1",
```
bump version

Bas950 reviewed (changes requested):
Upload proof of it working

VocalZero commented:
Changed schema version to 1.3 and updated presence version from 1.0.1 to 1.0.2

## Proof of work
![Screenshot 2021-03-16 235059](https://user-images.githubusercontent.com/20924469/111390580-40d68280-86b3-11eb-9c0e-1bb93e8b882b.png)
![Screenshot 2021-03-16 235146](https://user-images.githubusercontent.com/20924469/111390592-459b3680-86b3-11eb-8b23-19bf01db7974.png)
![Screenshot 2021-03-16 235205](https://user-images.githubusercontent.com/20924469/111390595-46cc6380-86b3-11eb-86d2-802c29cc6650.png)

InstabilSpielt commented:
Hello everyone, we sat down yesterday and thought about how we can ensure to avoid excessive requests even if no end timestamp of a song is available (For example, when we are currently live). Because we would normally trust socket.io, which unfortunately is not possible here, we decided to do it the oldschool way, with websockets.

VocalZero commented:
## Proof of work
![Screenshot 2021-03-19 025627](https://user-images.githubusercontent.com/20924469/111722795-d5331780-8862-11eb-9136-d31df3dbe7c9.png)

ririxidev commented on websites/A/atomicradio/dist/metadata.json:
```diff
@@ -1,23 +1,32 @@
+        "status.atomicradio.eu",
+        "docs.atomicradio.eu"
+    ],
+    "version": "1.0.2",
```
This update should be treated as 2.0.0

ririxidev commented on websites/A/atomicradio/presence.ts:
```diff
@@ -39,6 +110,27 @@ function clearPresenceData() {
+    return;
+  }
+
+  if (window.location.origin !== "https://atomicradio.eu") {
+    clearPresenceData();
+    return;
+  }
```
I don't see sense in this if statement because you only have 2 subdomains and one main domain.

ririxidev commented on websites/A/atomicradio/presence.ts:
```diff
@@ -39,6 +110,27 @@ function clearPresenceData() {
 }
 
 presence.on("UpdateData", async () => {
+  if (window.location.origin === "https://status.atomicradio.eu") {
+    clearPresenceData();
+    presenceData.details = "Viewing the current status";
+    presenceData.startTimestamp = browsingTimestamp;
+    presence.setActivity(presenceData, true);
+    return;
+  }
+
+  if (window.location.origin === "https://docs.atomicradio.eu") {
```
Instead of checking origin you could just check subdomain by ``window.location.host.split('.')[0]`` it will be easier and cleaner.

ririxidev reviewed (changes requested):
You should check my requested changes

Thanks,
ririxi (Contact: mail@ririxi.dev, ririxi#2721)

VocalZero commented:
Implemented your suggested changes

ririxidev reviewed (dismissed):
LGTM!

Bas950 commented on websites/A/atomicradio/dist/metadata.json:
```diff
@@ -1,23 +1,32 @@
+		"music",
+		"radio"
+    ],
+    "category": "music"
```
```suggestion
    "category": "music",
    "button": false
```

VocalZero commented:
Changes to the `metadata.json` have been applied.

ririxidev reviewed (approved):
LGTM ~~(again)~~!

Bas950 reviewed (approved):
LGTM
